bls/src/pod.rs

```diff
@@ -24,12 +33,27 @@ impl Default for SignatureCompressed {
     }
 }
 
+impl fmt::Display for SignatureCompressed {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        write!(f, "{}", BASE64_STANDARD.encode(self.0))
+    }
+}
+
+impl_from_str!(
+    TYPE = SignatureCompressed,
+    BYTES_LEN = BLS_SIGNATURE_COMPRESSED_SIZE,
+    BASE64_LEN = 128
```

Comment:
Do you think we should define 128 and 256 as constants as well?

Reply:
Oh oops, I meant to define these, but I must have missed it.